airflow/providers/google/cloud/operators/cloud_storage_transfer_service.py

```diff
@@ -1054,7 +1066,7 @@ def _create_body(self) -> dict:
             STATUS: GcpTransferJobsStatus.ENABLED,
             TRANSFER_SPEC: {
                 GCS_DATA_SOURCE: {BUCKET_NAME: self.source_bucket},
```

Comment:
Also here, if we're going to modify this operator to accept a destination path, should we add a `source_path` argument as well?

Reply:
Done, similarly as for the above S3 -> GCS